paper-dropdown-menu.html

```diff
@@ -390,15 +399,17 @@
          * optional `label` property.
          */
         _selectedItemChanged: function(selectedItem) {
-          var value = '';
+          var value = '',
+              label = '';
           if (!selectedItem) {
```

Comment:
I'm not sure I understand the point of what looks like a no-op in the `if(!selectedItem)`. Could be cleaned as follows:
```
if (selectedItem) {
  label = selectedItem.label || selectedItem.textContent.trim();
  ...
}
```